code/game/objects/items/stacks/money.dm

```diff
@@ -93,6 +105,33 @@
 	amount = 1
 	value = 1
 
+/obj/item/stack/money/rubles/New()
+	if(amount < 50)
```

Comment:
This is incredibly performance tasking; use this instead:

switch(amount)
if(0 to 49)
icon_state = "ruble"
else if(50 to 99)
icon_state = "ruble50"
else if(100 to 249)
icon_state = "ruble100"
else if(250 to INFINITY) (continue without infinity if etc)
icon_state = "ruble500"